Add PERIAN Job Platform Agent example (#1746)
flyteorg/flytesnacks: examples/perian_agent/README.md

# PERIAN Job Platform Agent

> Flyte Agent plugin for executing Flyte tasks on PERIAN Job Platform (perian.io). PERIAN allows the serverless execution of any task on servers aggregated from multiple cloud providers.

Reviewer: ```suggestion
The PERIAN Flyte Agent enables you to execute Flyte tasks on the [PERIAN Sky Platform](https://perian.io/). PERIAN allows the execution of any task on servers aggregated from multiple cloud providers.
```

Author: Much better :) updated

Example usage:

```{auto-examples-toc}
@task(container_image=image_spec,
      task_config=PerianConfig(
          accelerators=1,
          accelerator_type="A100",
      ))
def perian_hello(name: str) -> str:
    return f"hello {name}!"
```

To get started with PERIAN, see the [PERIAN documentation](https://perian.io/docs/overview) and the [PERIAN Flyte Agent documentation](https://perian.io/docs/flyte-getting-started).

> ## Agent Setup

Reviewer: ```suggestion
## Agent setup
```
